### DISCUSSION TITLE
Render custom component

### DISCUSSION THREAD

**Discussion by vxfil** [recharts/recharts#2734] [Q&A] [answered]
Is it possible to render a similar component to the left of each bar?
![image](https://user-images.githubusercontent.com/46137927/145851937-e8abf028-022c-4ab0-98ac-306f54e3bfa2.png)
if possible, please tell me how, thanks in advance)

**Comment by amapic** [comment, marked as answer]
Perhaps you can introduce a svg trough a react element in the label attribute of each bar

**Reply by vxfil** [reply]
thanks for the answer, I succeeded doing this with the help of LabelList
